lab-michael/lib/bodyParser.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+  // Will parse the JSON body
+
+module.exports = function(req, cb, err) {
```

Comment:
It's "usually" good practice for the callback to the be last parameter. Not a big deal, but just keep it in mind.